README.md

```diff
@@ -24,7 +24,7 @@ please ping the Oversight Committee for help.
   out from maintainers of relevant components. The broader the scope or more
   controversial the change, the more broad the consensus should be required for
   merging. The final approval and merge (or action, e.g. deleting a repo)
-  falls to a member of the Oversight Committee. This final review is to ensure that
+  falls to a member of the Oversight Committee, 2 Maintainers of any InstructLab Organization repository, or any 2 Organization owners. This final review is to ensure that
```

Comment:
Suggestion: Remove oversight committee member from this list of approvers (1 committee member != two maintainers but the OC committee members can certainly chime in). And add a follow-on sentence that if there is contention with a given PR that can't be resolved by maintainers, the oversight committee should be consulted as an arbitrator.

Note that (from my understanding) the purpose of the committee was to make sure the rules were followed and sufficient review was given (meaning 2 approvals may not be enough for some submissions).  So we might want to change the wording to not just be two approvals from maintainers, but instead have two maintainer approvals + for whoever presses the merge button, they are signing off that sufficient review was given.

Reply:
this makes sense to me! I will do this as well as removing the idea of "org owners" from the list of people who have these permissions